zebrad/src/components/mempool/crawler/tests.rs

```diff
@@ -21,31 +22,60 @@ const CRAWL_ITERATIONS: usize = 4;
 /// Note that this affects the total run time of the [`crawler_requests_for_transaction_ids`] test.
 /// There are [`CRAWL_ITERATIONS`] requests that are expected to not be sent, so the test runs for
 /// at least `MAX_REQUEST_DELAY * CRAWL_ITERATIONS`.
-const MAX_REQUEST_DELAY: Duration = Duration::from_millis(250);
+const MAX_REQUEST_DELAY: Duration = Duration::from_millis(25);
```

Comment:
25ms might be too short on heavily loaded VMs, but let's see how it goes.

Reply:
Oops, I changed during testing but maybe I shouldn't have committed it :( But it doesn't matter really, during the test the time is paused and it automatically advances to the next wake time whenever it becomes idle. So "in theory", it should finish as fast as possible as if every sleep was instantaneous. This is also why I had to make the Tokio executor single threaded. I suspect that with the multi-thread executor it might advance time if the main thread is idle, even if the workers are still executing things :/